lib/manageiq/providers/amazon/engine.rb

```diff
@@ -1,4 +1,5 @@
-require 'rails/all'
+require 'rails/engine'
+require 'action_controller/railtie' # needed for isolate_namespace below
```

Comment:
How does https://github.com/ManageIQ/manageiq-automation_engine/blob/master/lib/manageiq/automation_engine/engine.rb get away without requiring that railtie?